book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Good job setting up associations 💯, however, let's make sure a method for adding rentals is present. just taka an example form this:
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```